ibm_catalog.json

```diff
@@ -0,0 +1,102 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-ibm-icd-elasticsearch",
+        "label": "Databases-for-Elasticsearch",
+        "product_kind": "solution",
+        "tags": [
+          "ibm_created",
+          "target_terraform",
+          "terraform",
+          "databases",
+          "solution"
+        ],
+        "keywords": [
+          "elasticsearch",
+          "IaC",
+          "infrastructure as code",
+          "terraform",
+          "solution",
+          "elasticsearch secure",
+          "database",
+          "nosql"
+        ],
+        "short_description": "Creates and configures an instance of IBM Cloud Databases for Elasticsearch.",
+        "long_description": "This architecture supports creating and configuring an IBM Elasticsearch instance with KMS encryption and Context Based Restriction rules.",
+        "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-icd-elasticsearch/blob/main/README.md",
+        "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-icd-elasticsearch/main/images/elasticsearch_icon.svg",
+        "provider_name": "IBM",
+        "features": [
+          {
+            "title": "Creates an instance of Databases for Elasticsearch",
+            "description": "Creates an IBM Elasticsearch resource instance on the IBM Cloud environment."
+          },
+          {
+            "title": "Creates Context Based Restriction rules",
+            "description": "Creates and configures Context Based Restriction rules."
+          },
+          {
+            "title": "Supports kms encryption",
+            "description": "Provides the kms encryption feature."
+          },
+          {
+            "title": "Supports autoscaling",
+            "description": "Provides the autoscaling feature."
+          },
+          {
+            "title": "Attaches access tags",
+            "description": "Attach access tags to the IBM Elasticsearch instance."
+          }
+        ],
+        "flavors": [
+          {
+            "label": "Secure",
+            "name": "secure",
+            "install_type": "fullstack",
+            "working_directory": "solutions/secure",
+            "compliance": {
+              "authority": "scc-v3",
+              "profiles": [
+                {
+                  "profile_name": "IBM Cloud Framework for Financial Services",
+                  "profile_version": "1.5.0"
+                }
+              ]},
+            "iam_permissions": [
+              {
+                "role_crns": [
+                  "crn:v1:bluemix:public:iam::::serviceRole:Administrator"
+                ],
+                "service_name": "databases-for-elasticsearch"
+              },
+              {
+                "role_crns": [
+                  "crn:v1:bluemix:public:iam::::serviceRole:Administrator"
+                ],
+                "service_name": "cbr"
```

Comment:
remove this, CBR support not being added in initial release

Reply:
As we are not supporting CBR now, then please check if `Editor` role is sufficient. 
As per the main module readme - you should have editor role.

Reply:
Changed to Editor role